samples/nrf5340/netboot/README.rst

```diff
@@ -1,88 +1,92 @@
+The Network Core Bootloader sample, also called B0n, is an immutable first-stage bootloader that can update the application firmware on the network core of the nRF5340 System on Chip (SoC).
 
 Overview
 ********
 
-The network core bootloader sample supports transporting an already verified and authenticated firmware upgrade from the application core flash to the network core flash, as well as performing flash protection.
+The Network Core Bootloader sample protects the flash memory areas allocated to both itself and the application running on the network core.
 
-This is accomplished by the following steps:
+You must use this sample as a child image of a :ref:`multi-image <ug_multi_image>` build, where MCUboot is enabled and there is a network core application.
+MCUboot verifies and shares with the network core bootloader any new network core application image received through a device firmware update (DFU) transport layer, like a serial or a Bluetooth LE connection.
+For this reason, without MCUboot, this sample does nothing other than launching directly the application.
 
-1. Lock the flash of the bootloader.
-     The bootloader sample locks the flash that contains the sample bootloader and its configuration.
+During the boot process, the Network Core Bootloader sample and MCUboot interact as follows:
+
+1. MCUboot writes the Network Core Bootloader sample configuration to the shared SRAM.
+     In case it received a new network core application image, the configuration includes an update instruction and both the SHA and the data of the image.
+#. The bootloader sample locks the flash memory areas containing itself and its configuration.
      Locking is done using the ACL peripheral.
      For details on locking, see the :ref:`fprotect_readme` driver.
+#. The sample performs any pending network core firmware update.
+     The sample calls the :ref:`subsys_pcd` library to inspect the SRAM region shared with the application core:
 
-#. Perform any pending network core firmware upgrade.
-     Invoke the :ref:`subsys_pcd` library to inspect a SRAM region shared with the application core.
-     If the application core has written an update instruction, copy the specified data range to the application partition on the network core.
-     Once the copy is done, compare the SHA of the data in the application partition against the SHA specified in the shared SRAM.
-     Communicate the result of the comparison to the application core through the shared SRAM.
-
-#. Lock the flash of the application.
-     Lock the flash that contains the application.
+     a. If the application core has written an update instruction, the bootloader copies the specified data range to the application partition on the network core.
+     #. Once the copy is done, the sample compares the SHA of the data in the application partition against the SHA specified in the shared SRAM.
```

Comment:
Again, "the sample" here, but "the bootloader" above.

Reply:
Well, I didn't want to repeat the bootloader all the time. Anyway, I changed these mentions in the entire section. Fixed.